scripts-dev/complement.sh

```diff
@@ -11,12 +11,10 @@ cd "$(dirname $0)/.."
 docker build -t matrixdotorg/synapse:latest -f docker/Dockerfile .
```

Comment:
Is there a way to speed up this part of the Docker build?

Anytime you edit something in Synapse, this part takes 50 seconds to run: [`pip install --prefix="/install" --no-warn-script-location         /synapse[all]`](https://github.com/matrix-org/synapse/blob/eec9ab32253e458f275ffbc22368442b7ec5b549/docker/Dockerfile#L54-L55)

Reply:
This isn't the ideal way to do it, but we should track that outside of this PR.

Reply:
Split out into https://github.com/matrix-org/synapse/issues/9364